features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -278,7 +278,9 @@
 
     <feature name="openhab-binding-groheondus" description="GROHE ONDUS Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.groheondus/${project.version}</bundle>
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">wrap:mvn:org.grohe/ondus-api/0.0.7$Bundle-Name=GROHE%20ONDUS%20Api&amp;Bundle-SymbolicName=org.grohe.ondus-api&amp;Bundle-Version=0.0.7</bundle>
```

Comment:
https://github.com/FlorianSW/grohe-ondus-api-java/blob/master/build.gradle#L60

I suggest you release a version of the binding that does not contain its dependencies (is not "fat") and instruct gradle to create a valid OSGi manifest file (there are plugins), so that you don't need "Bundle-SymbolicName" and all that stuff here.

Reply:
Even if, in my opinion, the maintainer of a 3rd party library should not be responsible for integrating libraries in another project (I already did that by bundling the dependencies into one single jar, as openHAB had no dependency management system I felt comfortable with), I now implemented the changes in the library and release a new version (0.0.9) with the requested changes.

I also changed the things in the feature.xml, and the verification now succeeds. However, I'll add a comment to the pushed changes, as there're things I don't understand why this needs to be done, and what this has to do with a "modern" build and dependency management system.

Reply:
It's just nice to provide a manifest.mf file to be OSGi compatible. Helps projects like this one. 

Fat libraries like your last version are not necessary though in a maven world. 

You are probably complaining because you have not yet used OSGi. There are compile time dependencies (provided via pom) and runtime dependencies, provided by the OSGi container (karaf in this case). That's actually a quite good dependency management or at least there are no alternatives.

Reply:
> You are probably complaining because you have not yet used OSGi

That's absolutely possible and is most likely case. Please don't see that as general complaining or something like that. However, the ecosystem, especially in the open source world, shows, that OSGi is far away from being as popular as other (maybe even less flexible) dependency management systems.

I, for myself, just don't see the big advantages and benefits of having such a complicated setup here :)